cmd/openshift-install/create.go

```diff
@@ -128,3 +140,75 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		return nil
 	}
 }
+
+// FIXME: pulling the kubeconfig and metadata out of the root
+// directory is a bit cludgy when we already have them in memory.
+func destroyBootstrap(ctx context.Context, directory string) (err error) {
+	config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(directory, "auth", "kubeconfig"))
+	if err != nil {
+		return err
```

Comment:
`errors.Wrap` where useful in this func.

Reply:
> `errors.Wrap` where useful in this func.

Done with 43f08d1 -> 00fa119, which also drops [your `errors.Cause`][1] to print the error with its wrapping context.  Are you ok with that?

[1]: https://github.com/openshift/installer/commit/dc118f204223f3a2ef1239aa648f3a8745e51f94#diff-0aa4e247497495ef8506ad3e14deada6R28